hw/ip/entropy_src/dv/cov/entropy_src_UNR.vRefine

```diff
@@ -24,12 +25,15 @@
     <rule ccType="inst" ccfFlagsMask="2199136515010-42" domain="icc" entityName="entropy_src/29" entityType="block" excTime="1665673899" file="0" im-checksum="786255092" line="110" name="exclude" reviewer="unknown" text="assign intr_es_fatal_err_o          = Stub ? '0                 : core_intr_es_fatal_err;" user="1" vscope="default"></rule>
     <rule ccType="inst" ccfFlagsMask="2199136515010-42" domain="icc" entityName="entropy_src/32" entityType="block" excTime="1665673903" file="0" im-checksum="786255092" line="111" name="exclude" reviewer="unknown" text="assign alert_test                   = Stub ? stub_alert_test    : core_alert_test;" user="1" vscope="default"></rule>
     <rule ccType="inst" ccfFlagsMask="2199136515010-42" domain="icc" entityName="entropy_src/35" entityType="block" excTime="1665673907" file="0" im-checksum="786255092" line="112" name="exclude" reviewer="unknown" text="assign alert                        = Stub ? stub_alert         : core_alert;" user="1" vscope="default"></rule>
-    <rule ccType="inst" domain="icc" entityName="entropy_src/&quot;tl_o.d_param&quot;/2" entityType="toggle-bit" excTime="1665674018" excludeKind="rise" name="exclude" reviewer="unknown" user="1" vscope="default"></rule>
-    <rule ccType="inst" domain="icc" entityName="entropy_src/&quot;tl_o.d_param&quot;/1" entityType="toggle-bit" excTime="1665674018" excludeKind="rise" name="exclude" reviewer="unknown" user="1" vscope="default"></rule>
-    <rule ccType="inst" domain="icc" entityName="entropy_src/&quot;tl_o.d_param&quot;/0" entityType="toggle-bit" excTime="1665674018" excludeKind="rise" name="exclude" reviewer="unknown" user="1" vscope="default"></rule>
-    <rule ccType="inst" domain="icc" entityName="entropy_src/&quot;tl_o.d_sink&quot;/0" entityType="toggle-bit" excTime="1665674022" excludeKind="rise" name="exclude" reviewer="unknown" user="1" vscope="default"></rule>
-    <rule ccType="inst" domain="icc" entityName="entropy_src/&quot;entropy_src_xht_o.active&quot;/0" entityType="toggle-bit" excTime="1665674041" excludeKind="fall" name="exclude" reviewer="unknown" user="1" vscope="default"></rule>
+    <!-- Exclude toggle coverage of TL ports and alert_rx_i[0] input -->
+    <rule ccType="inst" domain="icc" entityName="entropy_src/&quot;tl_i.a_param&quot;" entityType="toggle" excTime="1668165562" name="exclude" reviewer="unknown" user="1" vscope="default"></rule>
```

Comment:
Can we make a separate Refine file for these?  I presume the motivation is here is that we understand that the unit TB cannot (and need not) hit these.

All the others are caught by `--cov-unr`.   If we make another run of cov-unr (which as @vogelpi points out we should, and I agree) do you think we'll have any trouble tracking the changes?

Reply:
This should have already been excluded with the common Xcelium coverage exclusion tcl script located at:
https://github.com/lowRISC/opentitan/blob/a8755be792fed312dc2ee0bb7095d35355a585f5/hw/dv/tools/xcelium/common_cov_excl.tcl#L6